orc8r/cloud/go/services/orc8r_worker/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package orc8r_worker contains the orc8r_worker service.
+//
+// The orc8r_worker service is a singleton service that runs different classes.
+// It currently only runs the reindexer as a singleton.
+
+
+package orc8r_worker
```

Comment:
Attach the doc comments directly to the pkg
```suggestion
// Package orc8r_worker contains the orc8r_worker service.
//
// The orc8r_worker service is a singleton service that runs different classes.
// It currently only runs the reindexer as a singleton.
package orc8r_worker
```